src/components/DetailsInfo/detailsInfo.util.js

```diff
@@ -206,13 +206,23 @@ export const generateAlertsDetailsInfo = pageData => {
       notificationsDetailsInfo: [],
       triggerCriteriaDetailsInfo: []
     }
-    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((valueItem, index) => (
-      <Tooltip
-        key={valueItem.tooltip + index}
-        template={<TextTooltipTemplate text={valueItem.tooltip} />}
-      >
-        {valueItem.icon}
-      </Tooltip>
+    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((notification, index) => (
+      <li className="notifications-item" key={index}>
+        <div className="notifications-item_icon">{notification.icon}</div>
+        <div>
+          <div className="notifications-item-header">{upperFirst(notification.kind)}</div>
+          <div className="notifications-item-header-text">
+            {`${notification.succeeded} success `}
+            <span
+              className={classNames('notifications-item-header-text_failed', {
```

Comment:
no need to use {}, you can add notification.failed > 0 && notifications-item-header-text_success